lib/internal/source_map/source_map.js

```diff
@@ -140,6 +142,9 @@ class SourceMap {
     }
     this.#payload = cloneSourceMapV3(payload);
     this.#parseMappingPayload();
+    if (ArrayIsArray(lineLengths) && ArrayLength(lineLengths)) {
```

Comment:
`array.length` is an own property of an `Array` instance and `Array.prototype.length` is not a method or getter. So there is no such primordial method `ArrayLength`. 

```suggestion
    if (ArrayIsArray(lineLengths) && lineLengths.length) {
```

The source map support would be disabled if there are any exceptions occurred when constructing it: https://github.com/nodejs/node/blob/main/lib/internal/source_map/prepare_stack_trace.js#L70. Running node with the environment variable `NODE_DEBUG=source_map` can reveal the internal logs of such exceptions.

Reply:
Perfect, thanks.